Userland/Libraries/LibGfx/TextLayout.cpp

```diff
@@ -21,14 +21,14 @@ RefPtr<GlyphRun> shape_text(FloatPoint baseline_start, Utf8View string, Gfx::Fon
 
     u32 glyph_count;
     auto* glyph_info = hb_buffer_get_glyph_infos(buffer, &glyph_count);
```

Comment:
Do we still need this call to `hb_buffer_get_glyph_infos`? Seems we never use the result of it.

Reply:
I don't understand harfbuzz very well, but I thought it does some additional parsing, *especially* because it is the call that sets `glyph_count` to 2 (instead of 1, like the first call), so it seems like it's necessary.

In any case, I would like to claim that "optimizing the code" is out of scope for "fixing the crash", at least in my eyes. Do you agree?

Reply:
Sure